### ISSUE TITLE
YTR 202 | Card reveal animation

### ISSUE THREAD

**Issue by cdanfon** [mozilla/foundation.mozilla.org#9272]
Nice to have 

ACs:

- [ ] As a user when I scroll down past the black panel (after Sign the petition) the content of the white panel is fixed until the black banner is out of view
- [ ] As a user when I scroll down past the white panel (after the timeline) the content of the black panel is fixed until the white banner is out of view

Example: https://www.abstract.com/state-of-design

**Comment by cdanfon** [comment]
We flagged during stand up yesterday that is very likely this nice to have won't be completed during the sprint

**Comment by siimonevans** [comment]
Confirmed to have been completed as part of [#9263](https://github.com/mozilla/foundation.mozilla.org/issues/9263).

This ticket can be progressed once the timeline is merged.

**Comment by siimonevans** [comment]
Reassigned to Steve for now as this is linked to the timeline work.
